java/test/jmri/jmrit/display/layoutEditor/LayoutEditorTest.java

```diff
@@ -24,6 +24,37 @@
 
     @Rule
     public RetryRule retryRule = new RetryRule(2); // allow 2 retries
+    
+    @BeforeClass
+    static public void setUpClass() {
+          JUnitUtil.setUp();
+    }
+
+    @AfterClass
+    static public void tearDownClass() {
+          JUnitUtil.tearDown();
```

Comment:
You need to remove this tearDownClass method.  This executes once after all tests in the file and is just repeating work that is done in TearDown after each test.

Reply:
There is no TearDown after each test.Should I change this to @After to do that instead?

Reply:
The tearDown routine started on line 51 of the file before you modified this.